examples/lessons/03/app.js

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable array-bracket-spacing, no-multi-spaces */
 import {
-  GL, AnimationLoop, Program, Model, Geometry, Matrix4,
-  resetParameters, setParameters
+  GL, AnimationLoop, Model, Geometry, Matrix4, setParameters, ShaderCache
```

Comment:
Very good idea: to make sure shader cache is used by one of the examples. This will help us catch regressions!

Although, I would suggest that we do this in Lesson 9 instead of this lesson. The reason is that lesson 9 creates 50 copies of a `Star` model and I can see an almost second long delay when I start it, so the introduction of the shaderCache in that example would be justified (I would expect it to completely eliminate the startup delay), and could be accompanied with a comment explaining the optimization technique. Could even deserve a bullet in the little HTML panel.

Lesson 3 is supposed to be a minimal example introducing only small incremental concepts, so I think it would be better to leave the shader cache out of this example.

Reply:
Agree, we should keep lesson 03 to show minimal increments, will remove it from there. In lesson 09 we create `Program` object once and use it all copies of `Star` model, so shaders are compiled only once. Only other better place I see is `picking` example, which creates multiple `Sphere` models, will add `shaderCache` there.